source/Preferences.cpp

```diff
@@ -32,7 +32,7 @@ namespace {
 	const string EXPEND_AMMO = "Escorts expend ammo";
 	const string FRUGAL_ESCORTS = "Escorts use ammo frugally";
 	
-	const vector<double> ZOOMS = {.06, .09, .13, .18, .25, .35, .50, .70, 1.00, 1.40, 2.00};
+	const vector<double> ZOOMS = {.06, .09, .13, .18, .25, .35, .50, .71, 1.00, 1.41, 2.00};
```

Comment:
I'm guessing Michael's original intention was to make them multiples of five. https://github.com/endless-sky/endless-sky/commit/436d345cb9a2b9a1a9990574ab16013d8155923d#commitcomment-19254321 might be relevant.